deploy/FlapperInit.sol

```diff
@@ -84,24 +113,16 @@ library FlapperInit {
         require(pairDai == dai,           "Dai mismatch");
         require(pairGem == flapper.gem(), "Gem mismatch");
 
-        require(cfg.hop >= 5 minutes, "hop too low");
         require(cfg.want >= WAD * 90 / 100, "want too low");
-        require(cfg.hump > 0, "hump too low");
 
-        flapper.file("hop",  cfg.hop);
         flapper.file("want", cfg.want);
         flapper.file("pip",  cfg.pip);
-        flapper.rely(address(dss.vow));
-        flapper.rely(address(mom));
+        flapper.rely(cfg.splitter);
 
-        dss.vow.file("flapper", address(flapper));
-        dss.vow.file("hump",    cfg.hump);
-        dss.vow.file("bump",    cfg.bump);
+        SplitterLike(cfg.splitter).file("flapper", flapper_);
 
-        mom.setAuthority(dss.chainlog.getAddress("MCD_ADM"));
-
-        dss.chainlog.setAddress("MCD_FLAP",    address(flapper));
-        dss.chainlog.setAddress("FLAPPER_MOM", address(mom));
+        if (cfg.prevChainlogKey != bytes32(0)) dss.chainlog.removeAddress(cfg.prevChainlogKey);
```

Comment:
```suggestion
        if (cfg.prevChainlogKey != bytes32(0) && cfg.prevChainlogKey != cfg.chainlogKey) dss.chainlog.removeAddress(cfg.prevChainlogKey);
```

Reply:
Not really sure is needed though.

Reply:
I'd expect `cfg.prevChainlogKey` to be set to `bytes32(0)` in case an existing chainlog key is reused. Worse case if it's set to non-zero by mistake, it will just remove and re-add the address (inefficient but still correct final state).